unitex/src/fr/umlv/unitex/leximir/delas/EditorDelas.java

```diff
@@ -0,0 +1,1364 @@
+            ad.setVisible(true);
+        } else {
+            JOptionPane.showMessageDialog(null, "No selected value");
+        }
+    }
+
+    private void jMenuViewMouseClicked(java.awt.event.MouseEvent evt) {
+        if (this.getjTable1().getSelectedRow() != -1) {
+            Object[] obj = new Object[8];
+            for (int i = 0; i < 8; i++) {
+                obj[i] = this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), i);
+            }
+            MenuDelas ad = new MenuDelas(this, this.getjTable1().getSelectedRow(), "view", this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), 7), obj);
+            ad.setVisible(true);
+        } else {
+            JOptionPane.showMessageDialog(null, "No selected value");
+        }
+    }
+
+    private void jMenuEditMouseClicked(java.awt.event.MouseEvent evt) {
+        if (this.getjTable1().getSelectedRow() != -1) {
+            Object[] obj = new Object[8];
+            for (int i = 0; i < 8; i++) {
+                obj[i] = this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), i);
+            }
+            MenuDelas ad = new MenuDelas(this, this.getjTable1().getSelectedRow(), "edit", this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), 7), obj);
+            ad.setVisible(true);
+        } else {
+            JOptionPane.showMessageDialog(null, "No selected value");
+        }
+    }
+
+    private void jMenuBeforeMouseClicked(java.awt.event.MouseEvent evt) {
+        if (this.getjTable1().getSelectedRow() != -1) {
+            Object[] obj = new Object[8];
+            for (int i = 0; i < 8; i++) {
+                obj[i] = this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), i);
+            }
+            MenuDelas ad = new MenuDelas(this, this.getjTable1().getSelectedRow(), "copyBefore", this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), 7), obj);
+            ad.setVisible(true);
+        } else {
+            JOptionPane.showMessageDialog(null, "No selected value");
+        }
+    }
+
+    private void jMenuAfterMouseClicked(java.awt.event.MouseEvent evt) {
+        if (this.getjTable1().getSelectedRow() != -1) {
+            Object[] obj = new Object[8];
+            for (int i = 0; i < 8; i++) {
+                obj[i] = this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), i);
+            }
+            MenuDelas ad = new MenuDelas(this, this.getjTable1().getSelectedRow(), "copyAfter", this.getjTable1().getValueAt(this.getjTable1().getSelectedRow(), 7), obj);
+            ad.setVisible(true);
+        } else {
+            JOptionPane.showMessageDialog(null, "No selected value");
+        }
+    }
+
+    private void jMenuSaveMouseClicked(java.awt.event.MouseEvent evt) {
+        int dialogResult = JOptionPane.showConfirmDialog(null, "This will overwrite your dictionaries. Are you sure?", "Save Delas Dictioneries in Unicode", JOptionPane.YES_NO_OPTION);
```

Comment:
There is a typo here.
It should be "Dictionaries".
The same applies at line 883.